src/SpriteSheet.js

```diff
@@ -50,13 +53,15 @@ export default class SpriteSheet extends React.PureComponent {
     this.time = new Animated.Value(0);
     this.interpolationRanges = {};
 
-    let { source, height, width, rows, columns } = this.props;
+    let { source, height, width, rows, columns, frameHeight, frameWidth, offsetY = 0, offsetX = 0 } = this.props;
     let image = resolveAssetSource(source);
     let ratio = 1;
     let imageHeight = image.height;
     let imageWidth = image.width;
-    let frameHeight = image.height / rows;
-    let frameWidth = image.width / columns;
+    offsetX = -offsetX;
+    offsetY = -offsetY;
```

Comment:
Why not just mention in the docs that offsetX and offsetY have to be negative or positive?

Reply:
Having it act like css offsets seems more natural imo

Reply:
fair